app/main.py

```diff
@@ -1,2 +1,72 @@
+from typing import Any
+
+
 class Dictionary:
-    pass
+    class Node:
```

Comment:
Consider making the `Node` class a private inner class by prefixing its name with an underscore, i.e., `_Node`. This will encapsulate it within the `Dictionary` class and indicate that it is intended for internal use only.